.babelrc

```diff
@@ -2,7 +2,7 @@
   "stage": 0,
   "env": {
     "development": {
-      "plugins": ["react-transform"],
+      "plugins": ["react-transform::after"],
```

Comment:
Shouldn't this...

```js
"plugins": ["react-transform::after"],
```

use a single colon, instead of two colons?

```js
"plugins": ["react-transform:after"],
```

See examples in the referenced discussion thread.